trie/verkle.go

```diff
@@ -163,16 +160,11 @@ func (t *VerkleTrie) UpdateAccount(addr common.Address, acc *types.StateAccount,
 		stem      = t.pointCache.GetTreeKeyBasicDataCached(addr[:])
 	)
 
+	binary.BigEndian.PutUint32(basicData[utils.BasicDataCodeSizeOffset:], uint32(codelen))
 	binary.BigEndian.PutUint64(basicData[utils.BasicDataNonceOffset:], acc.Nonce)
 	// get the lower 16 bytes of water and change its endianness
 	balanceBytes := acc.Balance.Bytes()
-	for i := 0; i < 16 && i < len(balanceBytes); i++ {
-		basicData[utils.BasicDataBalanceOffset+i] = balanceBytes[i]
-	}
-	// var cs [8]byte
-	// binary.BigEndian.PutUint64(cs[:], uint64(codeLen))
-	// copy(basicData[utils.BasicDataCodeSizeOffset:], cs[:3])
-
+	copy(basicData[32-len(balanceBytes):], balanceBytes[:])
```

Comment:
In [this comment](https://github.com/gballet/go-ethereum/pull/472/files#r1723369847) you mention that the balance switch to big endian is still pending.

Doesn't that mean that this code is incorrect? Since `Bytes()` returns a big endian?

Reply:
I pushed a fix to use big endian this morning, so now it's correct